Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -3719,9 +3719,15 @@ public void ResetRunnerExecutionDetails(bool doNotResetBusFlows=false)
             PublishToALMConfig = null;
             if (doNotResetBusFlows == false)
             {
-                foreach (BusinessFlow bf in BusinessFlows)
+                foreach (BusinessFlow businessFlow in BusinessFlows)
                 {
-                    bf.Reset();
+                    businessFlow.Reset();
+
+                    foreach (VariableBase inputVariable in businessFlow.GetBFandActivitiesVariabeles(true, true))
```

Comment:
why not to include it inside bf.Reset() function?

Reply:
bf.Reset() is called from many places including from Automate tab. 
But i want to reset variable value on run set start.